.changeset/huge-baths-tell.md

```diff
@@ -0,0 +1,7 @@
+---
+'skuba': minor
+---
+
+deps: Jest 30
+
+This major release includes breaking changes. See the [Jest 30](https://jestjs.io/blog/2025/06/04/jest-30) announcement for more information.
```

Comment:
Should we call out any impactful changes here?

Changes to `expect` aliases and snapshots could be worth mentioning.